modules/tinymce/src/core/main/ts/SelectionOverrides.ts

```diff
@@ -284,7 +283,8 @@ const SelectionOverrides = (editor: Editor): SelectionOverrides => {
           e.preventDefault();
           clipboardData.clearData();
           clipboardData.setData('text/html', realSelectionElement.outerHTML);
-          clipboardData.setData('text/plain', realSelectionElement.outerText);
+          // outerText is a nonstandard property and doesn't exist on Firefox, so fallback to innerText
+          clipboardData.setData('text/plain', (realSelectionElement as any).outerText || realSelectionElement.innerText);
```

Comment:
Bug fixed here, as Firefox just returns undefined unlike all other browsers (the new types picked this up).